server.js

```diff
@@ -9,7 +10,8 @@ var app          = express();
 app.set('views', `${__dirname}/app`);
 app.set('view engine', 'ejs');
 
-var apiUrl      = process.env.API_URL || 'https://api.cbddev.xyz';
+var domains     =(process.env.DOMAINS  || 'cbddev.xyz').split(';').map(o=>o.trim()).filter(o=>!!o);
```

Comment:
I will rather through error than continue with cbddev.xyz if the domains are not set

Reply:
I agree